keep/api/core/db.py

```diff
@@ -415,6 +415,15 @@ def get_all_workflows(tenant_id: str) -> List[Workflow]:
         ).all()
     return workflows
 
+def get_all_workflows_yamls(tenant_id: str) -> List[str]:
```

Comment:
Why is it necessary? We have an endpoint where you can download the raw yaml.

Reply:
you had endpoint for one yaml, to fit with your convention of having seperate endpoints for 1  workflow and all workflows I added one for all yamls